doc/modules/ROOT/pages/python-runtime/knowledge-graph-embeddings.adoc

```diff
@@ -0,0 +1,283 @@
+= Knowledge graph embeddings
+
+Knowledge Graph Embeddings (KGE) refer to a family of algorithms designed to learn low-dimensional representations of entities and relations within a knowledge graph.
+These embeddings are utilized for tasks including link prediction, entity classification, and entity clustering.
+
+This chapter provides an overview of the available shallow embedding algorithms in the GDS Python runtime, such as `TransE` and `DistMult`.
+
+* `TransE`
+** https://proceedings.neurips.cc/paper_files/paper/2013/file/1cecc7a77928ca8133fa24680a88d2f9-Paper.pdf[Bordes, Antoine, et al. "Translating embeddings for modeling multi-relational data." Advances in neural information processing systems 26 (2013).^]
+* `DistMult`
+** https://arxiv.org/pdf/1412.6575[Yang, Bishan, et al. "Embedding entities and relations for learning and inference in knowledge bases." arXiv preprint arXiv:1412.6575 (2014).^]
+
+A knowledge graph is a directed, multi-relational graph.
+It consists of triples `(head, relation, tail)` where `head` and `tail` are entities and `relation` is the relationship between them.
+
+Shallow models of KGE represented here are _inductive_ algorithms for computing node and relationship embeddings (low-dimensional representations) for knowledge graphs.
+Shallow means that matrix lookups represent the entity and relation encoders.
+The algorithm is inductive, meaning that it can be trained on one graph and then applied to this exact graph or another graph with the same schema.
```

Comment:
KGE models are not inductive. You get node embeddings as the output that only pertain the graph it was trained on